src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -210,12 +218,14 @@ async def post(self):
         tags:
           - Garden
         """
-        garden = GardenSchema(strict=True).loads(self.request.decoded_body).data
+        garden = self.validate_or_raise(
```

Comment:
This looks like it's calling the old form of `validate_or_raise` rather than your updated one (it's still taking a `method`).